hoprd-node-1.yaml

```diff
@@ -9,6 +9,8 @@ spec:
   version: riga
   secret:
     secretName: hoprd-ku5wzs
+  monitoring:
```

Comment:
Is this the corect amount of tabs?

Reply:
Yes, in next enhancements we can modify monitoring parameters like timeInterval, relabel_config, etc.